config/config.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+type Config struct {
+	Name		string			`mapstructure:"name"`
+	CliRoot	string			`mapstructure:"homeclient"`
+	Init		*InitConfig		`mapstructure:"init"`
+	GenTx		*GenTxConfig	`mapstructure:"gentx"`
```

Comment:
what's up with this formatting?

Reply:
looks like go-fmt is not enabled? Bunch of stuff is misformatted within this file